### PR TITLE
update membrane file plugin dependency

### PR DESCRIPTION
**Pull request by kevinschweikert**
Hey,

I currently try to build a pipeline where I need the file plugin 0.15 with the `seekable?: true` feature, which is only available in the newest version. This package depends on an older version, so I bumped the dependency version and updated the `ex_doc` and `membrane_file_plugin` versions.

The tests run still fine, so I thought this could be helpful to you.

Cheers
Kevin

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -48,7 +48,7 @@ defmodule Membrane.FFmpeg.SWResample.Mixfile do
       {:bundlex, "~> 1.0"},
       # Testing
       {:mockery, "~> 2.1", runtime: false},
-      {:membrane_file_plugin, "~> 0.14.0"},
+      {:membrane_file_plugin, "~> 0.15.0"},
```
Comment (mat-hek) [inline mix.exs:51]:
```suggestion
      {:membrane_file_plugin, "~> 0.15.0", only: :test},
```

**Review by mat-hek — APPROVED** [review]

**Comment by mat-hek** [comment]
Hi @kevinschweikert , thanks for updating it. Let's make it a test-only dependency, as the file plugin is only used in tests, and we can merge ;)

**Comment by kevinschweikert** [comment]
Would it make sense to also mark them as optional, so when you use this package in another mix project, the version is not relevant? like this:

```elixir
      # Testing
      {:mockery, "~> 2.1", runtime: false, only: :test, optional: true},
      {:membrane_file_plugin, "~> 0.15.0", only: :test, optional: true},
```

**Comment by mat-hek** [comment]
I believe `only: :test` works that way - now when you have `membrane_ffmpeg_swresample_plugin` as a dependency, it should work no matter what version of `membrane_file_plugin` you have in your project. Regarding mockery, AFAIK it cannot be made test only, unfortunately.

**Comment by kevinschweikert** [comment]
Thanks for the clarification. I updated my branch to mark the dependency as a test dependency. Do you want me to open another PR for this?

**Comment by mat-hek** [comment]
I've already marked it, I thought it's no point in making you do this. Sorry for the confusion. The plugin is already released as 0.17.3 ;)

**Pull request merged**